custom/icds_reports/tests/agg_tests/__init__.py

```diff
@@ -33,16 +34,17 @@ def setUpModule():
     setup_location_hierarchy(domain.name)
 
     with override_settings(SERVER_ENVIRONMENT='icds', STATIC_DATA_SOURCE_PROVIDERS=[]):
-        setup_tables_and_fixtures(domain.name)
-        aggregate_state_form_data()
-        try:
-            with mock.patch('custom.icds_reports.tasks.update_aggregate_locations_tables'):
-                move_ucr_data_into_aggregation_tables(datetime(2017, 5, 28), intervals=2)
-            build_incentive_report(agg_date=datetime(2017, 5, 28))
-        except Exception as e:
-            print(e)
-            tearDownModule()
-            raise
+        with flag_enabled('ICDS_LOCATION_REASSIGNMENT_AGG'):
```

Comment:
maybe is fine, but does this mean we will no longer have test coverage for our production agg until this gets rolled out?

Reply:
Yeah, it sort of a pick your poison situation, either the new agg won't have tests or the old one won't, but the risk that new changes break this logic seems higher than the old logic, and beyond the source tables, they are basically identical processes, so new changes shouldn't exercise the differences anyway.